### PR TITLE
#171: Contains comparison supports arrays

### PR DESCRIPTION
**Pull request by MartinGotelli**
This PRs fixes "bug" #171.
Contains comparison now support lists and tuples

### CODE REVIEW

File: django_mock_queries/comparisons.py [hunk 1]
```diff
@@ -10,6 +10,9 @@ def iexact_comparison(first, second):
 
 
 def contains_comparison(first, second):
+    if isinstance(first, list) or isinstance(first, tuple):
```
Comment (stefan6419846) [inline django_mock_queries/comparisons.py:13]:
```suggestion
    if isinstance(first, (list, tuple)):
```

**Comment by stefan6419846** [comment]
Could you please fix the linting issue which currently leads to CI failure?

**Comment by MartinGotelli** [comment]
> Could you please fix the linting issue which currently leads to CI failure?
Done, sorry, wasn't understanding why the checks were failing

**Comment by stefan6419846** [comment]
Thanks. No worries - I had to dig a bit into the log output as well to discover the actual issue ;)

**Pull request merged**